src/transformState.js

```diff
@@ -4,8 +4,33 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
-function transformState(state, actions) {
-  // write code here
+function transformState(states, actions) {
+  const state = states;
```

Comment:
I don't get it 🤔 
We're prompted to modify state, which is rarely the case but here we are 😄